mautic/mautic
# Fix reset date value from API

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Noticed we are not able to reset value of contact date/datetime value from API. This PR fixed it




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contact with custom date field and set the date
3. Create API  patch request to edit contact with set the date to empty and overwriteWithBlank to true (see image bellow)
4. Without fix date field was not reset, after fix should work properly

![image](https://user-images.githubusercontent.com/462477/106859884-f6311600-66c3-11eb-9551-0ed87efb11c3.png)
